src/longbow/filter/command.py

```diff
@@ -102,12 +94,29 @@ def main(pbi, out_prefix, model, force, input_bam):
             total=read_count
             ) as pbar:
 
+        # Get our model:
+        if model is None:
+            model = bam_utils.get_model_name_from_bam_header(bam_file.header)
+            lb_model = LibraryModel.build_pre_configured_model(model)
```

Comment:
same comment about reading from the json in the BAM header.

Reply:
Done